src/drivers/intel/hda/hda-dma.c

```diff
@@ -221,18 +221,6 @@ static void hda_dma_get_dbg_vals(struct dma_chan_data *chan,
 #define hda_dma_ptr_trace(...)
 #endif
 
-static void hda_dma_l1_entry_notify(void *arg, enum notify_id type, void *data)
```

Comment:
comment on commit message:

and the L1 mode will be re-entered after the L1 exit
done by toggling the SHIM_SVCFG_FORCE_L1_EXIT bit.

there is nothing in this patch that actually refers to this bit, and there is no explanation on when the L1 mode is re-entered, in other words you are saying both too much and too little at the same time.